tests/wpt/web-platform-tests/fetch/nosniff/stylesheet.html

```diff
@@ -3,28 +3,36 @@
 <script src=/resources/testharnessreport.js></script>
 <div id=log></div>
 <script>
-  var fails = ["", "?type=", "?type=x", "?type=x/x"],
-      passes = ["?type=text/css", "?type=text/css;blah"]
+  var fails = [null, "", "x", "x/x"],
+      passes = ["text/css", "text/css;charset=utf-8"]
```

Comment:
`text/css;blah` is equivalent of `text/css` once hyper has read it. This is actually why we didn't catch the problem with tests in the first place. Real content uses `text/css;charset=utf-8` and fails, current tests are not catching that. I can add the `blah` form if you really want it, but I think the test is just broken (they also failed to escape url parameters).